install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -26,22 +26,24 @@ spec:
       priorityClassName: "{{ .Values.global.priorityClassName }}"
 {{- end }}
       containers:
-        - name: validator
+        - name: galley
           image: "{{ .Values.global.hub }}/{{ .Values.image }}:{{ .Values.global.tag }}"
           imagePullPolicy: {{ .Values.global.imagePullPolicy }}
           ports:
           - containerPort: 443
           - containerPort: 9093
+          - containerPort: 9901
           command:
           - /usr/local/bin/galley
-          - validator
-          - --deployment-namespace={{ .Release.Namespace }}
           - --caCertFile=/etc/istio/certs/root-cert.pem
           - --tlsCertFile=/etc/istio/certs/cert-chain.pem
           - --tlsKeyFile=/etc/istio/certs/key.pem
-          - --healthCheckInterval=1s
-          - --healthCheckFile=/health
-          - --webhook-config-file
+          - --livenessProbeInterval=2s
```

Comment:
Can you restore the liveness and readiness probe intervals to their original values?

```
--readinessProbeInterval=1s
--livenessProbeInterval=1s

--interval=10s
--initialDelaySeconds: 5
periodSeconds=5
```

Reply:
@irisdingbj please address these with a new PR

Reply:
done